package.json

```diff
@@ -6,7 +6,7 @@
     "cypress:ci": "run-p --race storybook cypress:run",
     "cypress:dev": "run-p --race storybook cypress:open",
     "cypress:open": "cypress open",
-    "cypress:run": "wait-on http://localhost:9009 && cypress run --browser chrome",
+    "cypress:run": "wait-on http://localhost:9009/iframe.html && cypress run --browser chrome",
```

Comment:
http://localhost:9009 will return a loading bar now after upgrading storybook, so have to check the iframe.html instead

Reply:
kind of bummer, I missed it when storybook was a simple thing to use.